book/src/csi-driver-object.md

```diff
@@ -70,6 +75,19 @@ There are four important fields:
     The default is `Persistent`, which is the normal PVC/PV mechanism. `Ephemeral` enables
     [inline ephemeral volumes](ephemeral-local-volumes.md) in addition (when both
     are listed) or instead of normal volumes (when it is the only entry in the list).
+* `tokenRequests`
+  * This field was added in Kubernetes 1.20 and cannot be set when using an older Kubernetes release.
+  * This field is alpha.
+  * If this field is specified, Kubelet will plumb down the bound service acocunt tokens of the pod as `volume_context` in the `NodePublishVolume`:
+    * `"csi.storage.k8s.io/serviceAccount.tokens": {"gcp":{"token":"<token>","expirationTimestamp":"<expiration timestamp in RFC3339>"}}`
+    * If CSI driver doesn't find token recorded in the `volume_context`, it should return error in `NodePublishVolume` to inform Kubelet to retry.
+    * Audiences should be distinct, otherwise the validation will fail. If the audience is "", it means the issued token has the same audience as kube-apiserver.
+* `requiresRepublish`
+  * This field was added in Kubernetes 1.20 and cannot be set when using an older Kubernetes release.
+  * This field is alpha.
+  * If this field is `true`, Kubelet will periodically call `NodePublishVolume`. This is useful in the following scenarios:
+    * If the volume mounted by CSI driver is short-lived.
+    * If CSI driver requires valid service account tokens (enabled by the field `tokenRequests`) repeatedly.
```

Comment:
Can you add that note about how the republish should only update data, and not remount?

Reply:
done.